### PR TITLE
Support elm-sorter-experiment's Sort.Dict and Sort.Set

### PR DESCRIPTION
**Pull request by celsobonutti**

### CODE REVIEW

**Review by asterite — APPROVED** [review]

File: src/Elm/Decoder.hs [hunk 1]
```diff
@@ -4,17 +4,18 @@
...
-  , toElmDecoderSource
-  , toElmDecoderSourceWith
-  , renderDecoder
-  ) where
+  ( toElmDecoderRef,
+    toElmDecoderRefWith,
+    toElmDecoderSource,
+    toElmDecoderSourceWith,
+    renderDecoder,
+  )
```
Comment (charlieshanley) [inline src/Elm/Decoder.hs:12]: When possible, it's nice to put format chances in their own commit so they can be easily distinguished from "real" changes.

**Review by charlieshanley — APPROVED** [review]
Looks great! Thanks for doing this.

File: src/Elm/Decoder.hs (hunk 1 above)
Comment (celsobonutti) [reply src/Elm/Decoder.hs:12]: Oh, that was an accident, I didn't know which formatter this project used and then forgot to undo the changes my editor did automatically 😓

File: src/Elm/Decoder.hs [hunk 2]
```diff
@@ -139,6 +146,22 @@ instance HasDecoderRef ElmPrimitive where
     require "Dict"
     d <- renderRef (EList (ElmPrimitive (ETuple2 (ElmPrimitive key) value)))
     return . parens $ "map Dict.fromList" <+> d
+  renderRef (ESortDict sorter encoding key value) = do
+    require "Sort.Dict"
+    require "Sort.Dict.Extra" -- This is from our monolith, should we move it to an elm-package?
```
[the remark this answers is not in the document]
Comment (celsobonutti) [reply src/Elm/Decoder.hs:163]: Lets-a make it a library then 🎉
